packages/keycloak-user-management/src/components/UserList/index.tsx

```diff
@@ -113,11 +118,7 @@ const UserList = (props: Props): JSX.Element => {
       <h5 className="mb-3">User Management</h5>
```

Comment:
let's move this to a lang file, It will make it easy when the mls is implemented